front/app/modules/commercial/moderation/admin/containers/ModerationRow.tsx

```diff
@@ -145,36 +145,31 @@ const ModerationRow = memo<Props & InjectedIntlProps>(
       viewLink = `/${parentType.toLowerCase()}s/${parentSlug}`;
     }
 
-    const handleOnChecked = useCallback(
-      (_event: React.ChangeEvent) => {
-        onSelect(moderation);
-      },
-      [onSelect, moderation]
-    );
+    const handleOnChecked = (_event: React.ChangeEvent) => {
```

Comment:
On of the three useCallback instances here were causing a bug when a new post was flagged and you'd move to the Warnings tab (new tab in /admin/moderation) without refreshing the page. In the end, I didn't check which one it was specifically. Don't have the time at the moment to figure out if these `useCallback`s are worth it.

Reply:
I suspect handleOnChecked was causing the issue, since the others have no dependency on moderations.

Reply:
Once again, it sounds like missing dependencies to me. Maybe try and add them or remove the `useCallback`?
Can't wait for the linter update so that we can avoid these issues!

Reply:
The dependencies for this one seemed fine. The issue was a change of order in the calling of hooks (between useState and useCallback). So removing useCallback did the trick, although I didn't investigate time understanding why.